core/src/main/scala/com/salesforce/op/stages/impl/feature/Transmogrifier.scala

```diff
@@ -184,12 +185,18 @@ private[op] case object Transmogrifier {
         case t if t =:= weakTypeOf[TextAreaMap] =>
           val (f, other) = castAs[TextAreaMap](g)
           // Explicitly set cleanText to false here in order to match behavior of Text vectorization
-          f.vectorize(shouldPrependFeatureName = PrependFeatureName, cleanText = false, cleanKeys = CleanKeys,
+          f.smartVectorize(maxCategoricalCardinality = MaxCategoricalCardinality,
+            numHashes = DefaultNumOfFeatures, autoDetectLanguage = TextTokenizer.AutoDetectLanguage,
+            minTokenLength = TextTokenizer.MinTokenLength, toLowercase = TextTokenizer.ToLowercase,
+            prependFeatureName = PrependFeatureName, cleanText = false, cleanKeys = CleanKeys,
```

Comment:
Is there a reason why these weren't following the defaults in TransmogrifierDefaults in the first place? CleanText is set to true there.

Reply:
good point! it seems that this was an issue with vectorizer but smart vectorizer fixed it, fixed, thanks a lot!